ipaclient/csrgen_ffi.py

```diff
@@ -118,10 +149,7 @@
 char *ERR_error_string(unsigned long e, char *buf);
 ''')  # noqa: E501
 
-_libcrypto = _ffi.dlopen(ctypes.util.find_library('crypto'))
```

Comment:
Why are you removing this line? The code will break because we reference `_libcrypto` below.

Reply:
@abbra, thank you for looking at this. 
Actually, we need to know OpenSSL version ( `OpenSSL_version_num` / `SSLeay`) before parsing the related C code (`cdef`). Thus, the loading libcrypto in ABI mode happened earlier (see line 13, please).

Reply:
Ah, thanks. I missed that part. It looks OK to me but fedora-29/caless failed due to a timeout. I restarted the test.